scripts/pin_the_slow_way.py

```diff
@@ -34,7 +34,7 @@
           'ffmpeg': 'ffmpeg 2.8.*',
           'fontconfig': 'fontconfig 2.11.*',
           'freetype': 'freetype 2.6.*',
-          'geos: geos 3.4.*',
+          'geos': 'geos 3.4.*',
```

Comment:
Is it worth adding something to simply check for syntax errors on travis so that things like this don't leak through?